vortex-io/src/runtime/mod.rs

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: Copyright the Vortex contributors
+
+mod handle;
+pub use handle::*;
+#[cfg(not(target_arch = "wasm32"))]
+pub mod current;
+#[cfg(not(target_arch = "wasm32"))]
+pub mod multi;
+#[cfg(not(target_arch = "wasm32"))]
+pub mod single;
+#[cfg(not(target_arch = "wasm32"))]
+mod smol;
```

Comment:
Should the tokio and smol-specific things be behind feature flags?

Reply:
Ah yep, I'll do that in the next one